.cursor/rules/environment-variables.mdc

```diff
@@ -0,0 +1,107 @@
+---
+description: 
+globs: 
+alwaysApply: true
+---
+---
+description: Guidelines for handling environment variables in WAVS components
+globs: components/**/*.rs
+alwaysApply: true
+---
+# Environment Variable Handling
+
+Guidelines for managing environment variables in WAVS components:
+
+1. Required setup:
+```toml
+# In wavs.toml
+[service]
+name = "your-service"
+version = "0.1.0"
+
+[environment]
+# Public variables
+MAX_RETRIES = "3"
+TIMEOUT_SECONDS = "30"
+
+# Private variables (must be prefixed with WAVS_ENV_)
```

Comment:
@Reecepbcups is there a way we can set secrets in files that aren't committed to the repo?

Reply:
We might need to think more about what a good MVP around secrets management might look like.

Reply:
Noted. Docs updated.